server/src/test/scala/sbt/server/ProtocolTest.scala

```diff
@@ -621,13 +624,13 @@ class ProtocolTest {
           case ObjectToJson =>
             val json = addWhatWeWerePickling(t)(SerializedValue(t)(format.pickler))
             // IO.write(path, json.toString, IO.utf8)
-            if (path.exists) sys.error(s"$path exists already")
+            if (path.exists) () // Just ignore things we write twice, reading into different types should work.
```

Comment:
[eec6ad#R662](https://github.com/sbt/sbt-remote-control/commit/eec6ad2a22f5fae32587caee5ce3ece539ac5342#diff-33d6b80514066b409ffb90fb88c5c699R662):

```scala
+    oneWayTrip(Nil: List[String]) { _ / "array" / "nil.json" }
+    oneWayTrip(List("Bar", "Baz")) { _ / "array" / "seq_string.json" }
+    oneWayTrip(List(1, 2, 3)) { _ / "array" / "seq_int.json" }
+    oneWayTrip(List(true, false, true, true, false)) { _ / "array" / "seq_boolean.json" }
+    oneWayTrip(Vector("Bar", "Baz")) { _ / "array" / "seq_string.json" }
+    oneWayTrip(Vector(1, 2, 3)) { _ / "array" / "seq_int.json" }
+    oneWayTrip(Vector(true, false, true, true, false)) { _ / "array" / "seq_boolean.json" }
```

ok. We should make different dirs for List, Vector, and Seq.